starter-code/views/about.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+  <h1>About me</h1>
+
+  <p>Kyrie Andrew Irving (/ˈkaɪriː/; born March 23, 1992) is an American professional basketball player for the Boston Celtics of the National Basketball Association (NBA). He was named NBA Rookie of the Year after being selected by the Cleveland Cavaliers with the first overall pick in the 2011 NBA draft. A five-time NBA All-Star, Irving was selected to the All-NBA Third Team in 2015. He won an NBA championship with the Cavaliers in 2016.
+
+    Irving played college basketball for the Duke Blue Devils before joining the Cavaliers. He was named the NBA All-Star Game Most Valuable Player (MVP) in 2014. In the 2016 NBA Finals, he made a three-point field goal with 53 seconds remaining in a tied Game 7 to help lead the Cavaliers to a championship over the Golden State Warriors. After losing in the Finals in a rematch against the Warriors in 2017, Irving requested a trade, and he was dealt to Boston. Irving has also played for the United States national team, with whom he has won gold at the 2014 FIBA Basketball World Cup and the 2016 Summer Olympics.</p>
+
+    <a href="/about">Learn More About Me</a>
```

Comment:
This link is just taking the user back to the same page he is on, so is actually not very helpful :)